common/database.h

```diff
@@ -128,6 +128,7 @@ class Database : public DBcore {
 	bool SaveCharacterCreate(uint32 character_id, uint32 account_id, PlayerProfile_Struct *pp);
 	bool SetHackerFlag(const char *accountname, const char *charactername, const char *hacked);
 	bool SetMQDetectionFlag(const char *accountname, const char *charactername, const char *hacked, const char *zone);
+	bool SetMQDetectionFlag(const char *accountname, const char *charactername, const std::string &hacked, const char *zone);
```

Comment:
Can probably just remove the original method in favor of this one instead of overloading

Reply:
I left that just because, although I think removing it is probably fine. (There was some calls that use a string literal still, left it for them)